core/src/commonMain/kotlin/com/xebia/functional/xef/auto/DeserializerLLMAgent.kt

```diff
@@ -39,7 +39,7 @@ suspend inline fun <reified A> AIScope.prompt(
   user: String = "testing",
   echo: Boolean = false,
   n: Int = 1,
-  temperature: Double = 0.0,
+  temperature: Double = 0.5,
```

Comment:
Why is this change part of this PR?

Reply:
outdated changes, this branch needs to be merged with `main`. I'll update it.